lib/oli_web/components/delivery/course_content_panel.ex

```diff
@@ -1,26 +1,12 @@
 defmodule OliWeb.Components.Delivery.CourseContentPanel do
   use Phoenix.Component
 
-  import OliWeb.ViewHelpers
-
-  alias OliWeb.Router.Helpers, as: Routes
   alias OliWeb.Components.Delivery.CourseOutline
 
   def course_content_panel(assigns) do
     ~H"""
       <div class="container mx-auto mt-3 mb-5">
         <div class="bg-white dark:bg-gray-800 p-8 shadow">
-          <p class="text-secondary"><%= @description %></p>
```

Comment:
There's no longer any need for this Manage Section button with the new Manage tab, as it is redundant